File: Course1_Algorithmic_Toolbox/Assignments/Week2/lcm/lcm.cpp
#include <iostream>

long long lcm(int a, int b) {
  //write your code here
  return a*b;
}

int main() {
  int a, b;
  std::cin >> a >> b;
  std::cout << lcm(a, b) << std::endl;
  return 0;
}

File: Course1_Algorithmic_Toolbox/Assignments/Week2/lcm/LCM.java
import java.util.*;

public class LCM {
  private static long lcm(int a, int b) {
    //write your code here
    return a * b;
  }

  public static void main(String args[]) {
    Scanner scanner = new Scanner(System.in);
    int a = scanner.nextInt();
    int b = scanner.nextInt();

    System.out.println(lcm(a, b));
  }
}
